src/components/menu-options-button/menu-options-button.tsx

```diff
@@ -0,0 +1,55 @@
+import { styled } from "@mui/material/styles";
+import { theme } from "~/styles/theme";
+import { Button } from "@mui/base";
+import React, { type FC } from "react";
+import { SelectIcon } from "~/assets/icons/index.js";
+
+interface Props extends React.ComponentProps<typeof Button> {
+  text: string;
+}
+
+export const MenuOptionsButtonStyle = styled(Button)({
```

Comment:
Maybe put this in a separate `style.ts` file?

Reply:
For this particular case, The custom Buttons are more of a configuration. I'd rather keep them together in the same file